src/eventgrid/azext_eventgrid/aaz/latest/eventgrid/__cmd_group.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "eventgrid",
+    is_preview=True,
+)
+class __CMDGroup(AAZCommandGroup):
+    """Manage Azure Event Grid topics, domains, domain topics, system topics partner     topics, event subscriptions, system topic event subscriptions and partner topic event     subscriptions.
```

Comment:
subscriptions and partner topic event     subscriptions. --->
subscriptions, partner topic event subscriptions, and namespaces resources.